lib/plugins/aws/package/compile/events/cloudFront/index.js

```diff
@@ -206,16 +206,20 @@ class AwsCompileCloudFrontEvents {
               Object.assign(behavior, event.cloudFront.behavior);
             }
 
+            const lambdaFunctionAssociation = {
+              EventType: event.cloudFront.eventType,
+              LambdaFunctionARN: {
+                Ref: lambdaVersionLogicalId,
+              },
+            };
+
+            if (event.cloudFront.includeBody) {
```

Comment:
let's ensure we eventually pass _falsy_ value (as e.g. `false`), for that condition should be improve to 

```javascript
if (event.cloudFront.includeBody != null) {
```

Reply:
should I use `lambdaFunctionAssociation.IncludeBody = !!event.cloudFront.includeBody;` for adding the value to prevent non boolean value?